src/elements/Accordion/Accordion.js

```diff
@@ -122,13 +122,17 @@ class Accordion extends Component {
 			window.requestAnimationFrame(() => {
 				this.accordionContent.current.parentNode.style.height = 0;
 
+				const closeTransitionDuration = Math.max(this.accordionContent.current.clientHeight, 300) > 2000
+					? 1000
```

Comment:
this logic makes no sense
if it's gonna take longer than 2 seconds, it'll make it 1 second. else everything else can go up to 2.
both numbers should be 2000